patroni/ctl.py

```diff
@@ -268,7 +268,7 @@ def get_cursor(cluster, connect_parameters, role='master', member=None):
     return None
 
 
-def get_members(cluster, cluster_name, member_names, role, force, action):
+def get_members(cluster, cluster_name, member_names, role, force, action, scheduled_at):
```

Comment:
Lets change the signature to the following in order to avoid passing `None`.
```python
def get_members(cluster, cluster_name, member_names, role, force, action, scheduled_at=None):
```